packages/apollo-collaboration-server/src/fileHandling/fileHandling.service.ts

```diff
@@ -282,19 +281,15 @@ export class FileHandlingService {
    * @returns Return array of features (as JSON) if search was successful
    * or if search data was not found or in case of error return throw exception
    */
-  async getFeaturesByCriteria(searchDto: RegionSearchObjectDto) {
+  async getFeaturesByCriteria(searchDto: GFF3SequenceRegionDirective) {
```

Comment:
This should be `GFF3FeatureLine`

```suggestion
  async getFeaturesByCriteria(searchDto: GFF3FeatureLine) {
```